compiler/lib/backend.re

```diff
@@ -0,0 +1,175 @@
+open Stdlib;
```

Comment:
I was struggling a bit to implement some integration with the python backend. This should make that much easier!